src/components/sidebar-menu/SidebarMenu.styles.ts

```diff
@@ -0,0 +1,31 @@
+import palette from '~/styles/app-theme/app.pallete'
+
+export const styles = {
+  sidebarButton: {
+    width: '30%',
+    pr: { lg: '96px' },
```

Comment:
I think there should be other way to place items around. Padding is too big

Reply:
Thank you! I've delete this pudding
The space between boxes will be correct after adding page container width